fallout/jobs/job_types/legion/primelegionary.dm

```diff
@@ -0,0 +1,28 @@
+/datum/job/f13/legion/primelegionary
+	title = "Prime Legionary"
+	total_positions = 1
+	spawn_positions = 1
+	supervisors = "The Centurion, Prime and Veteran Decanii."
+	description = "Fight in the name of Caesar, and claim the lands for the bull! Listen to your superiors, They will guide you to victory."
+
+	outfit = /datum/outfit/job/f13/legion/primelegionary
+
+	display_order = JOB_DISPLAY_ORDER_F13_PRIMELEGIONARY
+
+/datum/outfit/job/f13/legion/primelegionary
+	name = "_Legion Veteran Legionary"
```

Comment:
Seems like a slip here. Prime Legionary with Vet Legionary job outfit name, actual loadout is fine though.